lottie/src/main/java/com/airbnb/lottie/utils/OffscreenLayer.java

```diff
@@ -553,4 +553,12 @@ private void renderHardwareShadow(Canvas targetCanvas, DropShadow shadow) {
     targetCanvas.drawRenderNode(shadowRenderNode);
     targetCanvas.restore();
   }
+
+  /**
+   * Whether calling finish() will decrement the save count of the [Canvas] returned by [start()].
+   */
+  public boolean finishDecrementsCanvasSaveCount() {
+    // endRecording() will decrement the save count of the Canvas returned by beginRecording()
+    return currentStrategy == RenderStrategy.RENDER_NODE;
```

Comment:
@geomaster I'm not sure why this happens, but I narrowed down the `beginRecording()`/`endRecording()` interaction as the root cause of the underflow error.